scripts/deploy

```diff
@@ -0,0 +1,189 @@
+#!/bin/bash
+
+build_chart() {
+  if [[ -z "${DOCKERHUB_USER}" ]]; then
```

Comment:
It's kinda weird to require DOCKERHUB_USER for build_chart, as there is no requirement for DockerHub for chart, we should logically separate this better.

Reply:
The dockerhub is used for the pushed image repository, I agree that its not something I'd Ideally like to require but it is documented in the help message and will be in the readme.